### PR TITLE
MC-13237: Added public port related fields and fixed the spacing issue.

### PR DESCRIPTION
**Pull request by y0geshdev**
#### Changes made
- Added the fields related to network policy rule which is to be created if specified fields are passed in create workload payload.
- Fixed some formatting issues as well.

### CODE REVIEW

**Review by danielvanacker — APPROVED** [review]

**Review by RGitB — APPROVED** [review]

File: source/includes/stackpath/_workloads.md [hunk 1]
```diff
@@ -220,6 +220,15 @@ Required | &nbsp;
...
  `secretEnvironmentVariableKey` <br/>*string* | The secret environment variable's key.
  `secretEnvironmentVariableValue` <br/>*string* | A sensitive environment variable that should not be exposed.
+ `deploymentInstancePerPops`<br/>*integer* | The number of deployments per point of presence. Only required if autoscaling is not enabled.
+ `cpuUtilization` <br/>*int* | Specify the percentage of CPU utilization.
+ `minInstancesPerPop` <br/>*int* | The minimum number of instances per PoP.
+ `maxInstancesPerPop` <br/>*int* | The maximum number of instances per PoP.
+ `publicPort`<br/>*string* | A single port, such as 80 or a port range, such as 1024-65535 for which a network policy rule will be created for the workload.
+ `publicPortSrc`<br/>*string* | A subnet that will define all the IPs allowed by the network policy rule.
+ `publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule.
+ `protocol`<br/>*string* | Protocol for the network policy rule. Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
```
Comment (razine-bensari) [inline source/includes/stackpath/_workloads.md:230]: I tried to create a workload without specifying the protocol and it is a required field. Should remove this field from the `Optional` and put it in the `Required`

File: source/includes/stackpath/_workloads.md [hunk 2]
```diff
@@ -118,12 +118,13 @@ Attributes | &nbsp;
...
 <!-------------------- CREATE A WORKLOAD -------------------->
+
 ### Create a workload
 ```shell
 curl -X POST \
     -H "MC-Api-Key: your_api_key" \
     -d "request_body" \
-    "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads"
+    "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/workloads"
 ```
```
Comment (razine-bensari) [inline source/includes/stackpath/_workloads.md:128]: Nice catch!

File: source/includes/stackpath/_workloads.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_workloads.md:230]: The issue is that if even one of these 4 fields is present then all of them are expected but these fields are optional in the sense that if we won't pass all 4 even then the workload will be created.
Due to this, it is unclear where to put these 4 fields.

**Review by razine-bensari — APPROVED** [review]
thks for the prompt changes

File: source/includes/stackpath/_workloads.md (hunk 1 above)
Comment (razine-bensari) [reply source/includes/stackpath/_workloads.md:230]: I gotchu! thks for the clarification!

Comment (y0geshdev) [reply source/includes/stackpath/_workloads.md:230]: I will see if I can add some sort of info section in API docs to mention that.

File: source/includes/stackpath/_workloads.md [hunk 3]
```diff
@@ -229,6 +229,10 @@ Required | &nbsp;
  `publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule.
  `protocol`<br/>*string* | Protocol for the network policy rule. Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
 
+ <aside class="notice">
+A workload can be created without `publicPort`, `publicPortSrc`, `publicPortDesc` and `protocol` as part of the payload. But all these fields are required to open a port/port-range on the created workload.
```
Comment (RGitB) [inline source/includes/stackpath/_workloads.md:233]: Nice addition 👌

File: source/includes/stackpath/_workloads.md [hunk 4]
```diff
@@ -220,6 +220,19 @@ Required | &nbsp;
...
+ `deploymentInstancePerPops`<br/>*integer* | The number of deployments per point of presence. Only required if autoscaling is not enabled.
+ `cpuUtilization` <br/>*int* | Specify the percentage of CPU utilization.
+ `minInstancesPerPop` <br/>*int* | The minimum number of instances per PoP.
+ `maxInstancesPerPop` <br/>*int* | The maximum number of instances per PoP.
+ `publicPort`<br/>*string* | A single port, such as 80 or a port range, such as 1024-65535 for which a network policy rule will be created for the workload.
+ `publicPortSrc`<br/>*string* | A subnet that will define all the IPs allowed by the network policy rule.
+ `publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule.
+ `protocol`<br/>*string* | Protocol for the network policy rule. Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
+
+ <aside class="notice">
```
Comment (y0geshdev) [inline source/includes/stackpath/_workloads.md]:
```suggestion
<aside class="notice">
```

File: source/includes/stackpath/_workloads.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_workloads.md:230]: 2aaca9e

**Pull request merged**